README.md

```diff
@@ -121,6 +121,7 @@ Parameters:
 * `targetElementSelector`: _string_, mandatory. The ID of the HTML tag used as target to place the Widget. It should be a CSS Path selector ID (_#IDSTRING_)
 * `options`: _object_, optional. Some global options available for all Widgets. Some specific Widgets could add mandatory option parametrizable using this `options` object. In this case, the options will be described in the [Widgets list](#widgets-list).
     - _`width`_: _mixte_, optional. If you want to force a specifc width for the Widget, indicate it here. It's taking the same type of value of the CSS width parameter: a specific width in pixel (`100px`) or in percentage (`50%`).
+    - _`height`_: _mixte_, optional. If you want to force a specifc height for the Widget, indicate it here. It's taking the same type of value of the CSS width parameter: a specific width in pixel (`100px`) or in percentage (`50%`).
```

Comment:
You forgot two instances of "width", they should be changed to "height" as well.

Also, will a height in percentage work? Doesn't it depends on the parent's height to be fixed?